### ISSUE TITLE
can you guide me how i can generate reid dataset from images.please 

### ISSUE THREAD

**Issue by Ritesh1991** [KaiyangZhou/deep-person-reid#454]

**Comment by siyanhu** [comment]
I'm just new to this library. So pls correct me if there's anything wrong. 
In make_db.py:
`class smu_sb_images(ImageDataset):
    dataset_dir = 'smu_sb_images'

    def __init__(self, root='', **kwargs):
        self.root = osp.abspath(osp.expanduser(root))
        self.dataset_dir = osp.join(self.root, self.dataset_dir)
        info = finder.get_image_info(self.dataset_dir)
        print(info)
        train = info
        query = info
        gallery = info
        super(smu_sb_images, self).__init__(train, query, gallery, **kwargs)`

after that in your train and testing file, e.g. mine is smu_sb_test.py:
`import torchreid
print(torchreid.models.show_avai_models())

from make_db import smu_sb_images
torchreid.data.register_image_dataset('smu_sb_images', smu_sb_images)

# 1. load data manager
datamanager = torchreid.data.ImageDataManager(
    root='reid-data',
    sources='cuhk03',
    targets='smu_sb_images',
    height=256,
    width=128,
    batch_size_train=32,
    batch_size_test=100,
    transforms=['random_flip', 'random_crop']
)


model = torchreid.models.build_model(
    name='resnet50',
    num_classes=datamanager.num_train_pids,
    loss='softmax',
    pretrained=True
)


weight_path = '/home/siyanhu/smartlight_reid/ops/model/osnet_x1_0_imagenet.pth'
torchreid.utils.load_pretrained_weights(model, weight_path)
model = model.cuda()


optimizer = torchreid.optim.build_optimizer(
    model,
    optim='adam',
    lr=0.0003
)

scheduler = torchreid.optim.build_lr_scheduler(
    optimizer,
    lr_scheduler='single_step',
    stepsize=20
)


# 3. build engine
engine = torchreid.engine.ImageSoftmaxEngine(
    datamanager,
    model,
    optimizer=optimizer,
    scheduler=scheduler,
    label_smooth=True
)


# 4. run train and test
engine.run(
    save_dir='log/resnet50',
    max_epoch=60,
    eval_freq=10,
    print_freq=10,
    test_only=True
)`
